### PR TITLE
hide var checking

### PR DESCRIPTION
**Pull request by mjudeikis**
:)

### CODE REVIEW

File: hack/create.sh [hunk 1]
```diff
@@ -14,16 +14,6 @@ if [[ -z "$AZURE_TENANT_ID" ]]; then
     exit 1
 fi
 
-if [[ -z "$AZURE_CLIENT_ID" ]]; then
```
Comment (kargakis) [inline hack/create.sh]: Why this diff? Can you just add the +x -x block?

Comment (mjudeikis) [reply hack/create.sh]: for all vars? I was thinking moving all sensitive data under one block just after this one

Comment (kargakis) [reply hack/create.sh]: Not all vars, just these two.

Comment (kargakis) [reply hack/create.sh]: Also I think we need to gate subscription and tenant id

**Comment by kargakis** [comment]
@stevekuznetsov @bbguimaraes these error messages don't make much sense - we need guidance on how to better structure our e2es in case this is an issue on our side or better insight into what's wrong.

**Comment by bbguimaraes** [comment]
Artifact collection needs to be enabled to get the logs from the failed pods, see:

https://github.com/openshift/release/tree/aad6ec8e8ac14e2f6a468e9de896bd142d309e74/ci-operator#using-a-template

e.g.

https://github.com/openshift/release/blob/aad6ec8e8ac14e2f6a468e9de896bd142d309e74/ci-operator/jobs/openshift/origin/openshift-origin-presubmits.yaml#L219-L225

**Pull request merged**